README.md

```diff
@@ -1,4 +1,4 @@
-# <img alt="NumPy" src="https://cdn.rawgit.com/numpy/numpy/master/branding/icons/numpylogo.svg" height="60">
+# <img alt="NumPy" src="https://ghcdn.rawgit.org/numpy/numpy/master/branding/icons/logomark/numpylogoicon.svg" height="60">
 
```

Comment:
This doesn't seem to be rendering properly, hit the ... button and choose View file. Maybe the image needs more height?

Reply:
I think there is a chicken-and-egg problem. The link is the post-PR place where the logo will be when the PR is merged. It currently lives in https://github.com/isabela-pf/numpy/blob/logorefresh/branding/icons/logomark/numpylogoicon.svg.

With that, the new logo is just the icon, where the old one was icon + text so I think we should add the text after it:
```
# <img alt="logo" src="https://ghcdn.rawgit.org/numpy/numpy/master/branding/icons/logomark/numpylogoicon.svg" height="60"> NumPy
```

Reply:
Hmm, that link is commented out, github seems to make its own link by default.

Reply:
You can also see the effect of changes when editing the file and using `preview`.

Reply:
I think in markdown, a line that begins with a single `#` is rendered as `<H1>`.

Reply:
Apparently you can link directly to files in the repo: https://stackoverflow.com/questions/14494747/add-images-to-readme-md-on-github

Reply:
Lets put this in and worry about the link later.

Reply:
I think this will work, but we can try later. The images don't seem to be rendered in the preview.
```
# <img alt="NumPy" src="https://github.com/numpy/numpy/blob/master/branding/icons/logomark/numpylogoicon.svg?raw=true" height="60">
```